bin/ta_puppet_report_viewer/modalert_generate_detailed_report_helper.py

```diff
@@ -126,14 +134,14 @@ def process_event(helper, *args, **kwargs):
     if is_json(event_dict["_raw"]):
         event_content = json.loads(event_dict["_raw"])
         event_uuid = event_content['transaction_uuid']
-        event_puppetdb_callback_hostname = event_content['puppetdb_callback_hostname']
+        event_pe_console = event_content['pe_console']
     else:
         event_uuid = None
-        event_puppetdb_callback_hostname = ""
+        event_pe_console = ""
 
     uuid = helper.get_param("transaction_uuid") or event_uuid
 
-    puppetdb_callback_hostname = helper.get_param("puppetdb_callback_hostname") or event_puppetdb_callback_hostname
+    pe_console = helper.get_param("pe_console") or event_pe_console
```

Comment:
Nit alert: In this section, for style purposes, the use of single vs double quotes should be consistent.